src/Surfnet/StepupMiddleware/ApiBundle/Controller/VerifiedSecondFactorController.php

```diff
@@ -104,4 +117,21 @@ public function collectionOfIdentityAction(Request $request)
 
         return JsonCollectionResponse::fromPaginator($paginator);
     }
+
+    public function getSkipVettingAction($id)
```

Comment:
Not sold on the name of this action. As this will return a boolean (json encoded) result, a method name starting with `is` seems more intuitive.

Some quick proposal

`isProvePossesionRequiredAction`
`isProvePossesionRequiredForAction`

Reply:
Changed to `getCanSkipProvePossession`